tests/basics/string_center.py

```diff
@@ -11,3 +11,5 @@
 print("foo".center(5))
 print("foo".center(6))
 print("foo".center(20))
+
+print(b"foo".center(20))
```

Comment:
please put this test in a separate file `bytearray_center.py`, so it can be automatically skipped if the port doesn't have `bytearray`

Reply:
I've just done that in the latest push, it's just a byte version I added left here how.  Is that better not having in a str_ test file?

Reply:
oh, sorry, I didn't see the update!

You'll see that there are specific tests for bytes methods, eg `bytes_strip.py`.  So it would be more consistent to have a new `bytes_center.py` test as well.